src/auth/auth.service.ts

```diff
@@ -1,100 +1,163 @@
-import { Injectable } from '@nestjs/common';
-import { InjectRepository } from '@nestjs/typeorm';
-import * as bcrypt from 'bcrypt';
-import { randomBytes } from 'crypto';
-import { User } from './user.entity';
-import { Repository } from 'typeorm';
-import * as jwt from 'jsonwebtoken';
-import { Uploads } from 'src/uploads/uploads.entity';
-import { MailService } from 'src/mail/mail.service';
-
+import { Injectable, UnauthorizedException } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import * as bcrypt from "bcrypt";
+import { randomBytes } from "crypto";
+import { User } from "./user.entity";
+import { MongoCompatibilityError, Repository } from "typeorm";
+import * as jwt from "jsonwebtoken";
+import { Uploads } from "src/uploads/uploads.entity";
+import { MailService } from "src/mail/mail.service";
 @Injectable()
 export class AuthService {
   constructor(
     @InjectRepository(User)
     private readonly userRepository: Repository<User>,
     @InjectRepository(Uploads)
     private readonly phonebookRepository: Repository<Uploads>,
-    private mailService: MailService,
-  ) { }
+    @InjectRepository(Uploads)
+    private readonly uploadsRepository: Repository<Uploads>,
+    private mailService: MailService
+  ) {}
 
   async login(email: string, password: string): Promise<any> {
+    try {
+      const Authuser = await this.userRepository.findOne({ where: { email } });
```

Comment:
follow a single naming convention across the project. Add a lint file to prevent such issues.